Fix the following errors observed in an x86 (32-bit) build:

  third_party/crashpad/crashpad/snapshot/x86/cpuid_reader.cc(116,3): error: always_inline function '_fxsave' requires target feature 'fxsr', but would be inlined into function 'SupportsDAZ' that is compiled without support for 'fxsr'
    116 |   _fxsave(&fxsave);
        |   ^

  third_party/crashpad/crashpad/snapshot/win/system_snapshot_win.cc(438,3): error: always_inline function '_fxsave' requires target feature 'fxsr', but would be inlined into function 'CPUX86SupportsDAZ' that is compiled without support for 'fxsr'
    438 |   _fxsave(&extended_registers);
        |   ^

Upstream bug report: https://issues.chromium.org/368113218

--- a/build/config/win/BUILD.gn
+++ b/build/config/win/BUILD.gn
@@ -130,6 +130,11 @@ config("compiler") {
       cflags += [ "-msse3" ]
     }
 
+    # Need FXSR instruction support for Crashpad on x86.
+    if (current_cpu == "x86") {
+      cflags += [ "-mfxsr" ]
+    }
+
     # Enable ANSI escape codes if something emulating them is around (cmd.exe
     # doesn't understand ANSI escape codes by default). Make sure to not enable
     # this if remoteexec is in use, because this will lower cache hits.
